pkg/rpc/client.go

```diff
@@ -58,7 +58,7 @@ type Provider interface {
 	// The method takes a context for cancellation and a slice of parameters to filter the vote accounts.
 	// It returns a pointer to a GetVoteAccountsResponse struct containing the vote accounts details,
 	// or an error if the operation fails.
-	GetVoteAccounts(ctx context.Context, params []interface{}) (*GetVoteAccountsResponse, error)
+	GetVoteAccounts(ctx context.Context, params []interface{}) (*VoteAccounts, error)
```

Comment:
I changed this as it is only the `GetVoteAccountsResponse.Result` which is actually used by the exporter